Task 3/Task3/QueueExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Task3
+{
+    public static class QueueExtensions
+    {
+        public static IQueue<T> Tail<T>(this IQueue<T> queue, int length = 10)
```

Comment:
In this implementation you change the original queue (as result it is empty). But this method has to create new queue not changing the original one